compiler/AST/checkAST.cpp

```diff
@@ -214,8 +216,6 @@ void checkPrimitives()
      case PRIM_ARRAY_SHIFT_BASE_POINTER:
      case PRIM_ARRAY_SET:
      case PRIM_ARRAY_SET_FIRST:
-     case PRIM_ERROR:
-     case PRIM_WARNING:
```

Comment:
I'm pretty sure I uncovered an existing problem. In other words, I think that on master, PRIM_WARNING (but not PRIM_ERROR - I included that for sanity reasons) can still exist after resolution. It's just that my PR calls classifyPrimitive in executeOnClauses on more functions than it used to (in order to figure out if they are local, not just fast). That means that we are asserting no PRIM_WARNING on some function we didn't used to check. My changes to checkAST and resolution are just to fix the problem.